snapshots/devmapper/metadata.go

```diff
@@ -48,13 +49,6 @@ var (
 	deviceIDBucketName = []byte("device_ids") // Tracks used device ids <device_id_[0..maxDeviceID)>=<byte_[0/1]>
 )
 
-var (
-	// ErrNotFound represents an error returned when object not found in meta store
-	ErrNotFound = errors.New("not found")
-	// ErrAlreadyExists represents an error returned when object can't be duplicated in meta store
-	ErrAlreadyExists = errors.New("object already exists")
-)
```

Comment:
Actually, one comment. There's a small chance, but just to be safe in case anyone was relying on these in some package, could you just assign these to errdefs.ErrNotFound and errdefs.ErrNotImplemented

Reply:
@dcantah 
Thanks for review that.
I think you point is good.
I had updated it..
thanks for that..

Reply:
We do not guarantee the Go API stability, and it is unlikely that somebody is depending on these variables. Also it is quite easy for such potential users to apply this change.